docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx

```diff
@@ -330,34 +309,51 @@ export default function RootLayout({ children }) {
+
+If you're using a global CSS file, import it into your `app/layout.tsx` file:
+
+```tsx filename="app/layout.tsx" switcher
+import '../index.css'
+
+// ...
+```
+
+If you're using Tailwind, you'll need to install `postcss` and `autoprefixer`:
+
+```bash filename="Terminal"
+npm install postcss autoprefixer
+```
+
+Then, create a `postcss.config.js` file at the root of your project:
 
-### Step 5: Create the Entrypoint Page
+```js filename="postcss.config.js"
+module.exports = {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
+  },
+}
+```
 
-On Next.js you declare an entrypoint for your application by creating a `page.tsx` file. The
-closest equivalent of this file on CRA is your `src/index.tsx` file. In this step, you’ll set up the
-entrypoint of your application.
+### Step 7: Create the Entrypoint Page
 
-1. **Create a `[[...slug]]` directory in your `app` directory.**
+On Next.js you declare an entrypoint for your application by creating a `page.tsx` file. The closest equivalent of this file on CRA is your `src/index.tsx` file. In this step, you’ll set up the entry point of your application.
 
-Since this guide is aiming to first set up our Next.js as an SPA (Single Page Application),
-you need your page entrypoint to catch all possible routes of your application. For that, create a
-new `[[...slug]]` directory in your `app` directory.
+**Create a `[[...slug]]` directory in your `app` directory.**
 
-This directory is what is called an
-[optional catch-all route segment](/docs/app/building-your-application/routing/dynamic-routes#optional-catch-all-segments).
-Next.js uses a file-system based router where
-[directories are used to define routes](/docs/app/building-your-application/routing/defining-routes#creating-routes).
-This special directory will make sure that all routes of your application will be directed to its
-containing `page.tsx` file.
+Since this guide is aiming to first set up our Next.js as an SPA (Single Page Application), you need your page entry point to catch all possible routes of your application. For that, create a new `[[...slug]]` directory in your `app` directory.
 
-2. **Create a new `page.tsx` file inside the `app/[[...slug]]` directory with the following content:**
+This directory is what is called an [optional catch-all route segment](/docs/app/building-your-application/routing/dynamic-routes#optional-catch-all-segments). Next.js uses a file-system based router where [directories are used to define routes](/docs/app/building-your-application/routing/defining-routes#creating-routes). This special directory will make sure that all routes of your application will be directed to its containing `page.tsx` file.
+
+**Create a new `page.tsx` file inside the `app/[[...slug]]` directory with the following content:**
 
 ```tsx filename="app/[[...slug]]/page.tsx" switcher
-import '../../src/index.css'
```

Comment:
Is there any reason to ditch `src`/ in particular or just personal preference?

Reply:
The docs instruct you to add the `app` folder inside `src`. So this path, `../../src/index.css`, errored.

Reply:
Ah yeah, this one could go either way. We could also say to keep src separate from app. I'm fine with it!